ci/sawtooth-build-docs

```diff
@@ -36,8 +36,10 @@ RUN echo "deb http://repo.sawtooth.me/ubuntu/ci xenial universe" >> /etc/apt/sou
     git \
     libffi-dev \
     libssl-dev \
+    libzmq3-dev \
```

Comment:
Is this related?

Reply:
Yes, It is needed for the rust to build correctly